src/main/java/at/gepardec/cditraining/interceptors/FirstInterceptor.java

```diff
@@ -1,22 +1,27 @@
 package at.gepardec.cditraining.interceptors;
 
+import org.slf4j.Logger;
 import org.slf4j.MDC;
 
 import javax.annotation.Priority;
+import javax.inject.Inject;
 import javax.interceptor.AroundInvoke;
 import javax.interceptor.Interceptor;
 import javax.interceptor.InvocationContext;
 
 @Interceptor
 @FirstIntercept
-@Priority(Interceptor.Priority.APPLICATION + 1)
```

Comment:
@cchet 
hier habe ich die priority gesetzt damit die interceptoren in der richtigen reihenfolge aufgerufen werden.
gibt es da eine andere möglichkeit?

Reply:
Siehe interceptors/interceptors.html

Reply:
mir kam es so vor, dass die reihenfolge dann random ist wenn keine priority gesetzt ist.

Reply:
Sollte die Reihenfolge der Declaration sein, war bei mir so. 
Das Problem ist was ist wenn von einer Lib Interceptoren kommen, die ordnet man die?
Da gefällt mir die Reihenfolge der Deklaration auf der Klasse mehr, dort sieht man sehr gut wie die geordnet sind.